tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -227,6 +227,32 @@ def fe_input_user_data_repack(
     else:
         assert input_user_shapes is None
     # TODO: implement freeze_placeholder (issue 58560)
```

Comment:
please remove this line because actually you are adding freezing support for all frontends